src/main/osd/osd_elements.c

```diff
@@ -211,7 +230,7 @@ static void renderOsdEscRpmOrFreq(getEscRpmOrFreqFnPtr escFnPtr, osdElementParms
         const int rpm = MIN((*escFnPtr)(i),99999);
         const int len = tfp_sprintf(rpmStr, "%d", rpm);
         rpmStr[len] = '\0';
-        displayWrite(element->osdDisplayPort, x, y + i, DISPLAYPORT_ATTR_NONE, rpmStr);
```

Comment:
Should everything go through `osdDisplayWrite()` now? If so we should remove `displayWrite()`.

Reply:
Yeah, probably sneaked in after I've created the dev branch.